forest/shared/src/lib.rs

```diff
@@ -3,3 +3,42 @@
 
 pub mod cli;
 pub mod logger;
+
+use std::path::PathBuf;
+
+#[cfg(feature = "rocksdb")]
+pub type Db = forest_db::rocks::RocksDb;
+
+#[cfg(feature = "paritydb")]
+pub type Db = forest_db::parity_db::ParityDb;
+
+/// Gets chain data directory
+pub fn chain_path(config: &crate::cli::Config) -> PathBuf {
+    PathBuf::from(&config.client.data_dir).join(&config.chain.name)
+}
+
+#[cfg(feature = "rocksdb")]
+/// Gets database directory
+pub fn db_path(config: &crate::cli::Config) -> PathBuf {
+    chain_path(config).join("rocksdb")
+}
+
+#[cfg(feature = "paritydb")]
+/// Gets database directory
+pub fn db_path(config: &crate::cli::Config) -> PathBuf {
+    chain_path(config).join("paritydb")
+}
```

Comment:
Maybe refactor this to a single function.

Reply:
done